plugin.video.catchuptvandmore/resources/lib/resolver_proxy.py

```diff
@@ -109,8 +111,46 @@ def get_stream_kaltura(plugin,
 def get_stream_dailymotion(plugin,
                            video_id,
                            download_mode=False):
-    url_dailymotion = URL_DAILYMOTION_EMBED % video_id
-    return get_stream_default(plugin, url_dailymotion, download_mode)
+
+    # TODO reactivate when youtubedl is fixed
```

Comment:
youtubedl was upadted recently. Can you provide a bit more info what's broken?

Reply:
It's not an issue of the Youtube-DL Kodi script, but an issue of Youtube-DL core itself.
Since 2020.03.24 version, the Dailymotion extractor of Youtube-DL do not work anymore with Python 2.
See https://github.com/ytdl-org/youtube-dl/issues/23256.
I opened a PR here https://github.com/ytdl-org/youtube-dl/pull/24776 but I am not sure I will be accepted...